File: C++11/백준/Bronze/32684. 장기/장기.cc
#include <stdio.h>

double score(double s){
    int b;
    scanf("%d",&b);
    s+=b*13;
    scanf("%d",&b);
    s+=b*7;
    scanf("%d",&b);
    s+=b*5;
    scanf("%d",&b);
    s+=b*3;
    scanf("%d",&b);
    s+=b*3;
    scanf("%d",&b);
    s+=b*2;
    return s;
}
int main(){
    double s1=0,s2=1.5;
    s1=score(s1);
    s2=score(s2);
    if(s1>s2){
        printf("cocjr0208");
    }
    else{
        printf("ekwoo");
    }
}

File: Java/백준/Bronze/32684. 장기/장기.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);double s1 = in.nextInt()*13+in.nextInt()*7+in.nextInt()*5+in.nextInt()*3+in.nextInt()*3+in.nextInt()*2;
        double s2 = in.nextInt()*13+in.nextInt()*7+in.nextInt()*5+in.nextInt()*3+in.nextInt()*3+in.nextInt()*2+1.5;
        if(s1>s2){
            System.out.println("cocjr0208");
        }
        else{
            System.out.println("ekwoo");
        }
    }
}